tests/playwright/shiny/bugs/1844-datagrid-empty-column/test_1844_datagrid_empty_column.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import annotations
+
+from playwright.sync_api import Page
+
+from shiny.playwright import controller
+from shiny.run import ShinyAppProc
+
+
+def test_row_selection(page: Page, local_app: ShinyAppProc) -> None:
```

Comment:
Thank you for the test!!

-----------------

```suggestion
def test_empty_column_name(page: Page, local_app: ShinyAppProc) -> None:
```